### PR TITLE
Assimilate braingames-libs

### LINKED ISSUE #2153: Move braingames-libs to webknossos repo
and update `Build.scala` to `build.sbt` in the process.

<a href="https://timer.scm.io/repos/537237ce1a00001a003c4a14/issues/2153/create?referer=github" target="_blank">Log Time</a>

### PR DESCRIPTION
**Pull request by fm3**
Includes the code from braingames-libs repository into webknossos.
There are now the projects `util`, `braingamesBinary`, `braingamesDatastore` and `webknossos`.

Note: is based on branch build-sbt, see PR #2257

`sbt run` starts webknossos, `sbt "project webknossosDatastore" run` starts standalone datastore.

Since there is no more braingames-libs version, the standalone datastore version is now the webknossos version, prefixed with "wk-" (now starts with wk-2.0.2 which corresponds to 11.3.10)

### Issues:
- fixes #2153

------
- [x] Ready for review

### CODE REVIEW

**Comment by normanrz** [comment]
The root folder feels a bit unorganized to me. 
- Can we merge `braingames-datastore` and `webknossos-datastore` into one? 
- Why does it make sense to split `braingames-binary` and `braingames-datastore`.
- Does `util` need its own directory?

File: build.sbt [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+lazy val webknossos = (project in file("."))
+  .dependsOn(util, braingamesBinary, braingamesDatastore)
+  .enablePlugins(play.sbt.PlayScala)
+  .enablePlugins(BuildInfoPlugin)
+  .settings((webknossosSettings ++ AssetCompilation.settings ++ BuildInfoSettings.webknossosBuildInfoSettings):_*)
+
+
+
+lazy val assemblyStandaloneDatastore = inputKey[Unit]("assembly standaloneDatastore")
```
Comment (normanrz) [inline build.sbt]: I don't think `assembly` is used anymore.

**Comment by fm3** [comment]
I think it makes sense to have util be a separate project since it does not semantically belong specifically to either webknossos or the datastore. This way, both can use its code.

I agree, we should try to merge braingames-binary, braingames-datastore and webknossos-datastore. I would suggest to do this in a separate PR, though, to keep the changes small, what do you think?

**Comment by normanrz** [comment]
I added CircleCI code for webknossos-datastore. 

Would the merge of the three projects be such a large task? I suppose the compiler will definitively tell us whether it worked?

**Comment by fm3** [comment]
I’ll have a look in the coming days

**Comment by jfrohnhofen** [comment]
the reason there are separate projects for `braingamesDatastore` and `webknossosDatastore` is, that one is a library / module, that is either used by wk or the other. I am not sure, this setup is still possible, when we merge both into one project.

**Comment by jfrohnhofen** [comment]
we should definitely merge `braingames-binary` and `braingames-datastore`, but I would also like to keep the `utils` separately.

**Pull request merged**